.eslintrc.js

```diff
@@ -109,7 +109,7 @@ module.exports = {
     // Custom errors:
     'no-use-before-define': [2, { 'functions': false, 'classes': true, 'variables': false }],
     'no-constant-condition': [2, { 'checkLoops': false }],
-    'new-cap': [2, { 'properties': false, 'capIsNewExceptionPattern': 'EAX|OCB|GCM|CMAC|CBC|OMAC|CTR', 'newIsCapExceptionPattern': 'type|hash*' }],
+    'new-cap': [2, { 'properties': false, 'capIsNewExceptionPattern': 'EAX|OCB|GCM|CMAC|CBC|OMAC|CTR|HKDF', 'newIsCapExceptionPattern': 'type|hash*' }],
```

Comment:
Very minor but HKDF is really just a function that computes the hash, whereas all the others are somewhat class-like in that they return functions, so I wouldn't appease the linter and just rename the function to `computeHKDF` or the likes.